### ISSUE TITLE
Multiple values in a tag should be separated by a comma

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#916]
_From [RossyMiles](https://code.google.com/u/107303732767247089612/) on October 24, 2010 11:57:10_

What steps will reproduce the problem? 1. Tag a song with multiple values in the Artist field. (like "Queen" and "David Bowie")
2. Play the song in Clementine. What is the expected output? What do you see instead? Foobar2000 and other players separate both values with a comma (Queen, David Bowie.) Clementine's interface and last.fm scrobbler just separates them with a space (Queen David Bowie.) This can be confusing and ambiguous. What version of the product are you using? On what operating system? Clementine r2182 on Arch Linux

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=916_

**Comment by bemeurer** [comment]
Please make this work. It's not a complex feature, and it greatly increases usability. The de-facto standard for field-separation is a semicolon, `;`, although perhaps interpreting both commas and semicolons wouldn't be bad.

**Comment by Moonbase59** [comment]
I agree with @eriteric: Separators for container formats that don’t support multiple tags should be user selectable. I use MusicBrainz Picard a lot, and they handle it well: User-define it for ID3 tags, use separate tags for FLAC. (I use semicolon+blank `; `, btw.)

**Comment by TheYokai** [comment]
Is there any progress being made toward this? A pr to contribute to?

This is a must have feature for me.

**Comment by jaksco** [comment]
I also use a semicolon ;);:);:D

**Comment by danifavi** [comment]
Very interested too! Clementine is great, but this functionality really needs to be implemented...

**Comment by TheYokai** [comment]
@danifavi I did some preliminary research into this and I feel like the database needs to be rethought in order to really get this going. Basically, we need a many-to-many connection for any arbitrary number of tags. While Clementines' database would need a bit of a refactor, it should be within the possibility space without completely breaking backward compatibility. I would be interested in hearing what @hatstand and other longer-term developers thought about making changes to the database system in order to support not only multiple tag values per track but also support any arbitrary number of tags a user wishes to apply.

**Comment by hatstand** [comment]
Backwards incompatible DB changes are fine but you do need to automatically migrate from old ones. This shouldn't be particularly hard in Clementine.

What might be a worry is performance when searching tags but it's probably fine.

**Comment by natrius** [comment]
Is this still available? It is possible to tag Genre automatically with Musicbrainz Picard and choose your own tag-separation. Right now i used Semikolon `;` without spaces for example because thats already supported by Foobar2000, my Windows Music-Player. I opened the same issue on the guayadeque github as well and i hope both players will support this feature. 

Maybe later the "occasion" or the "mood" will be tagged as well,. also with semikolon-separated values. (Or let users choose themself how the values are separated, similar to Musicbrainz Picard when tagging).

**Comment by alianza** [comment]
Also eagerly awaiting this!
